account_multi_foreign_vat/README.rst

```diff
@@ -0,0 +1,41 @@
+======================================
+Account Multi Foreign VAT
```

Comment:
README files in OCA are autogenerated from fragments, see https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

Reply:
Okay, I didn't know